xbmc/interfaces/legacy/RenderCapture.h

```diff
@@ -35,32 +34,36 @@ namespace XBMCAddon
 
     class RenderCapture : public AddonClass
     {
-      CRenderCapture* m_capture;
+      unsigned int m_captureId;
```

Comment:
@FernetMenta with those additions it works ... i needed to move the capture call to the player started callback. Because before we simply started capture when the addon was loaded ... but at that time there is no player and RenderCaptureAlloc would simply return NULL (and it would stay NULL for the addon forever).